README.md

```diff
@@ -44,9 +43,8 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     </tr>
     <tr>
-      <td align="center" valign="top" width="14.28%"><a href="https://adiati.com/"><img src="https://avatars.githubusercontent.com/u/45172775?v=4?s=100" width="100px;" alt="Ayu Adiati"/><br /><sub><b>Ayu Adiati</b></sub></a><br /><a href="#question-adiati98" title="Answering Questions">💬</a> <a href="#blog-adiati98" title="Blogposts">📝</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=adiati98" title="Code">💻</a> <a href="#content-adiati98" title="Content">🖋</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=adiati98" title="Documentation">📖</a> <a href="#example-adiati98" title="Examples">💡</a> <a href="#mentoring-adiati98" title="Mentoring">🧑‍🏫</a> <a href="#talk-adiati98" title="Talks">📢</a> <a href="#translation-adiati98" title="Translation">🌍</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/droffilc1"><img src="https://avatars.githubusercontent.com/u/97587370?v=4?s=100" width="100px;" alt="Clifford Mapesa"/><br /><sub><b>Clifford Mapesa</b></sub></a><br /><a href="#a11y-droffilc1" title="Accessibility">️️️️♿️</a> <a href="#tutorial-droffilc1" title="Tutorials">✅</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/edgarefigueroa"><img src="https://avatars.githubusercontent.com/u/55363462?v=4?s=100" width="100px;" alt="Edgar Figueroa"/><br /><sub><b>Edgar Figueroa</b></sub></a><br /><a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=edgarefigueroa" title="Code">💻</a> <a href="#data-edgarefigueroa" title="Data">🔣</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=edgarefigueroa" title="Documentation">📖</a> <a href="#research-edgarefigueroa" title="Research">🔬</a> <a href="#security-edgarefigueroa" title="Security">🛡️</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=edgarefigueroa" title="Tests">⚠️</a> <a href="#translation-edgarefigueroa" title="Translation">🌍</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/samgkigotho"><img src="https://avatars.githubusercontent.com/u/116047791?v=4?s=100" width="100px;" alt="Samgkigotho"/><br /><sub><b>Samgkigotho</b></sub></a><br /><a href="#blog-samgkigotho" title="Blogposts">📝</a> <a href="https://github.com/TejasQ/opensauced-guestbook/issues?q=author%3Asamgkigotho" title="Bug reports">🐛</a> <a href="#content-samgkigotho" title="Content">🖋</a> <a href="#data-samgkigotho" title="Data">🔣</a> <a href="#design-samgkigotho" title="Design">🎨</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=samgkigotho" title="Documentation">📖</a> <a href="#example-samgkigotho" title="Examples">💡</a> <a href="#ideas-samgkigotho" title="Ideas, Planning, & Feedback">🤔</a></td>
```

Comment:
It looks like you're overwriting Edgar's profile. Make sure that his remains and yours is separate.

Reply:
@samgkigotho this still needs to be resolved. But everything else looks good.

Reply:
I'm still not seeing Edgar here.

![image](https://github.com/open-sauced/guestbook/assets/34313413/3812e334-4e15-40a7-825f-342b03344c94)

Reply:
@samgkigotho did you see my comment above?

Reply:
Yes, I have tried to make the changes.